### ISSUE TITLE
[BUG]  Deprecation warning when build

### ISSUE THREAD

**Issue by Mayb3Nots** [jhomlala/betterplayer#439]
**History check**
Yes cant find any exsiting open issue.

**Describe the bug**
When building I get this.
```
C:\scr\flutter\.pub-cache\hosted\pub.dartlang.org\better_player-0.0.64\android\src\main\java\com\jhomlala\better_player\BetterPlayerPlugin.java:27: warning: [deprecation] Registrar in PluginRegistry has been deprecated
import io.flutter.plugin.common.PluginRegistry.Registrar;
                                              ^
C:\scr\flutter\.pub-cache\hosted\pub.dartlang.org\better_player-0.0.64\android\src\main\java\com\jhomlala\better_player\BetterPlayerPlugin.java:28: warning: [deprecation] FlutterMain in io.flutter.view has been deprecated
import io.flutter.view.FlutterMain;
```

**Expected behavior**
No warning logs

**Flutter doctor**
<details><summary>flutter doctor</summary>

```
[√] Flutter (Channel stable, 2.0.4, on Microsoft Windows [Version 10.0.18363.1500], locale en-MY)
[√] Android toolchain - develop for Android devices (Android SDK version 30.0.2)
[√] Chrome - develop for the web
[√] Android Studio (version 4.0)
[√] IntelliJ IDEA Community Edition (version 2020.3)
[√] VS Code (version 1.55.2)
[√] Connected device (4 available)
```
</details>

**Better Player version**
- Version: 0.0.62

**Comment by jhomlala** [comment]
I agree with you, this should be fixed. Will do it in next release(s).

**Comment by jhomlala** [comment]
Fixed in 0.0.65.
